Templates/PROJECT-TEMPLATE.md

```diff
@@ -1,11 +1,12 @@
 # CATEGORYXX-PROVIDERXXX - Project Title
 
-- ✍️ Title should be in the form of: CATEGORY01-PROVIDER100 - TITLE
+- ✍️ [Title should be in the form of: CATEGORY01-PROVIDER100 - TITLE]
   - `CATEGORY` is BIL, SEC, GOV, etc.
   - `01` is the subcategory seen in the main README
   - `PROVIDER` is AWS, AZ, GCP, OTH(other, please use an abbreviation)
   - `100` is both the level of difficulty as well as a unique ID of the project. (100,101,102 - see already existing projects)
   - `TITLE` is the title of the project. Be brief but informative.
+  - Make sure the filename matches the Title and is in the right folder, fe. `Projects/COM/COM03/COM03-AZ200.md`
```

Comment:
generally its eg or ie, but it doesn't matter to me lol

Reply:
Right, I never know how to use those haha, gonna stick to "eg." in the future👍